source/retryable-writes/tests/insertOne-serverErrors.yml

```diff
@@ -303,3 +302,170 @@ tests:
                 data:
                     - { _id: 1, x: 11 }
                     - { _id: 2, x: 22 }
+    -
+        description: "InsertOne succeeds after WriteConcernError InterruptedAtShutdown"
+        clientOptions:
+            retryWrites: true
+        failPoint:
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data:
+                failCommands: ["insert"]
+                writeConcernError:
+                    code: 11600
+                    errmsg: Replication is being shut down
+        operation:
+            name: "insertOne"
+            arguments:
+                document: { _id: 3, x: 33 }
+        outcome:
+            result:
+                insertedId: 3
+            collection:
+                data:
+                    - { _id: 1, x: 11 }
+                    - { _id: 2, x: 22 }
+                    - { _id: 3, x: 33 }
+    -
+        description: "InsertOne succeeds after WriteConcernError InterruptedDueToReplStateChange"
+        clientOptions:
+            retryWrites: true
+        failPoint:
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data:
+                failCommands: ["insert"]
+                writeConcernError:
+                    code: 11602
+                    errmsg: Replication is being shut down
```

Comment:
errmsg shouldn't be checked in this case. Should we leave it as an empty string?

Reply:
Agreed the errmsg shouldn't be checked but I think it's good to include a message since the server always(?) does. Also it's easier to debug:
```
>>> _raise_write_concern_error({"code":11602, "errmsg": ""})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "pymongo/helpers.py", line 208, in _raise_write_concern_error
    error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteConcernError
```
Vs:
```
>>> _raise_write_concern_error({"code":11602, "errmsg": "Replication is being shut down"})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "pymongo/helpers.py", line 208, in _raise_write_concern_error
    error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteConcernError: Replication is being shut down
```